cfme/tests/ansible/test_embedded_ansible_actions.py

```diff
@@ -63,18 +63,26 @@ def ansible_credential(wait_for_ansible, appliance, full_template_modscope):
         fauxfactory.gen_alpha(start="cred_"),
         "Machine",
         username=credentials[full_template_modscope.creds]["username"],
-        password=credentials[full_template_modscope.creds]["password"]
+        password=credentials[full_template_modscope.creds]["password"],
     )
     yield credential
 
     credential.delete_if_exists()
 
 
 @pytest.mark.tier(3)
-@pytest.mark.parametrize('create_vm_modscope', ['full_template'], indirect=True)
-def test_action_run_ansible_playbook_localhost(request, ansible_catalog_item, ansible_action,
-        policy_for_testing, create_vm_modscope, ansible_credential, ansible_service_request,
-        ansible_service):
+@pytest.mark.parametrize("create_vm_modscope", ["full_template"], indirect=True)
```

Comment:
When you apply black formatting to the module in a PR that is fixing the test, its hard to tell what is actually changing test behavior and what's just auto-formatted.

Please, in future PRs, refrain from including auto-formatting changes along with functional changes.

pre-commit config is excluding black formatting from most of the repo - if you're interested in including more of the repository in our auto-formatting, that's great! Just open separate PRs updating the pre-commit configuration for black, and applying it to the given directories.

Otherwise the next person that modifies this file won't be adhering to the same formatting as what you've applied here.